### DISCUSSION TITLE
control + f commands

### DISCUSSION THREAD

**Discussion by PoikaU** [Chatterino/chatterino2#4023] [Ideas] [answered]
Could you amazing hackerpeople make control + f work with commands. It gets quite annoying when you want to find some old pasta from 200+ commands. :) (xD)

**Comment by Felanbird** [comment]
Would require making triggers searchable, maybe also command responses? Either way probably a good addition to have more "settings" searchable.

**Comment by Mm2PL** [comment, marked as answer]
subset of https://github.com/Chatterino/chatterino2/issues/2674
